<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>783</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>图</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_play{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_play:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_play:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_play:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>Num</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_clear{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_clear:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_clear:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_clear:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DFS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>580</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>580</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dijkstra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_testPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>213</width>
      <height>228</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_di">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>560</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_flag">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_flag{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_flag:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_flag:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_flag:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Flag</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>1361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
